common_sensor_launch/launch/gscam.launch.py

```diff
@@ -0,0 +1,153 @@
+# Copyright 2022 Tier IV, Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from launch import LaunchDescription
+from launch.actions import DeclareLaunchArgument
+from launch.actions import OpaqueFunction
+from launch.conditions import LaunchConfigurationEquals
+from launch.conditions import LaunchConfigurationNotEquals
+from launch.substitutions import LaunchConfiguration
+from launch_ros.actions import ComposableNodeContainer
+from launch_ros.actions import LoadComposableNodes
+from launch_ros.descriptions import ComposableNode
+import yaml
+
+
+def launch_setup(context, *args, **kwargs):
+    # https://github.com/ros2/launch_ros/issues/156
+    def load_composable_node_param(param_path):
+        with open(LaunchConfiguration(param_path).perform(context), "r") as f:
+            return yaml.safe_load(f)["/**"]["ros__parameters"]
+
+    composable_nodes = [
+        ComposableNode(
+            package="gscam",
+            plugin="gscam::GSCam",
+            name=["gscam", LaunchConfiguration("camera_id")],
+            namespace="/sensing/camera/",
```

Comment:
@wep21 Can this namespace be specified in `aip_*` launch file?

Reply:
@aohsato No, do you have the case where you change the namespace?

Reply:
Thanks!

> do you have the case where you change the namespace?

I think yes, for example, just launching one camera. It's better to follow `common_sensor_launch` style.